.circleci/config.yml

```diff
@@ -0,0 +1,135 @@
+#
+# Copyright 2016-2018 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:9-jdk
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: maven-dependencies-{{ checksum "pom.xml" }}
+
+      - run:
+          name: Download dependencies
+          command: |
+            sudo apt-get install xsltproc
+            ./build-support/go-offline.sh
+      - save_cache:
+          key: maven-dependencies-{{ checksum "pom.xml" }}
+          paths:
+            - ~/.m2
+
+      - run:
+          name: Tests
+          command: ./mvnw clean verify
+
+      - run:
+          name: Collect test reports
+          command: |
+            mkdir -p /tmp/test-reports/unit-tests/
+            find . -type f -regex ".*/target/surefire-reports/.*xml" -exec cp {} /tmp/test-reports/junit/ \;
```

Comment:
`/tmp/test-reports/junit/` should be `/tmp/test-reports/unit-tests/`

Reply:
Same as https://github.com/openzipkin/zipkin-gcp/pull/85#discussion_r200860632, will do the same here as we agree on there.